packages/fixie-web/src/voice.ts

```diff
@@ -339,7 +341,7 @@ export class VoiceSession {
     } else if (msg.type === 'output') {
       this.handleOutputChange(msg.text, msg.final);
     } else if (msg.type === 'voice_synced_transcript') {
-      this.handleOutputTranscript(msg.text);
```

Comment:
should we just change this to make onOutputChange always expose the synced transcript, rather than having separate callbacks for slightly different transcripts?

I'd also prefer to retain the "final" argument to onOutputChange, which seems straightfoward to accommodate.

Reply:
I think it makes sense to remove callbacks related to LLM generation.  We can't have a "final" argument in the TTS transcript case until I implement that server-side, so my proposal would be to ship this, update the server to send {delta, final} instead of {text}, then update this to combine all the output-related callbacks into one with the full tts-synced transcript text and whether it's final.

Tangentially related, I'd love to pull this SDK into our monorepo once we're the only users.

Reply:
That plan sounds good.